### PR TITLE
Dx 1233/tn options

### PR DESCRIPTION
**Pull request by BryanZWu**

### CODE REVIEW

**Comment by BryanZWu** [comment]
Oops. Looks like something isn't building. Let me take a look.

File: lib/tnOption.js [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+  });
+
+  var request = client.createPostRequest(client.concatAccountPath(TN_OPTION_PATH), {tnOptionOrder: order});
+  request.buffer().then(res =>{
+    if(res.ok && res.headers.location){
+      Client.getIdFromLocationHeader(res.headers.location, function(err, id){
+        if(err){
+          return callback(err);
+        }
+        TnOption.get(client, id, callback);
```
Comment (jmulford-bw) [inline lib/tnOption.js:71]: I don't think this call is necessary. Why can't you just return the response from the post request?

Comment (BryanZWu) [reply lib/tnOption.js]: That's weird. I saw the other parts of the SDK doing it this way, so I just figured there was a reason for it. Fixed.

Comment (jmulford-bw) [reply lib/tnOption.js]: ah that's strange, yeah I don't think that's necessary

**Comment by BryanZWu** [comment]
Think that should be it. Let me know if there's anything else!

**Pull request merged**